src/Database/LSMTree/Internal.hs

```diff
@@ -1091,8 +1101,9 @@ open sesh label snap = do
       let runPaths = V.map (bimap (second $ RunFsPaths (Paths.activeDir $ sessionRoot seshEnv))
                                   (V.map (RunFsPaths (Paths.activeDir $ sessionRoot seshEnv))))
                            runNumbers
-      with (openLevels hfs hbio (confDiskCachePolicy conf) runPaths)
-           (newWithLevels sesh seshEnv conf)
+      with (openLevels hfs hbio (confDiskCachePolicy conf) runPaths) $ \lvls -> do
+        am <- newArenaManager -- TODO: bracketOnError
```

Comment:
You're right -- I'll remove it. At the time that I wrote this, I thought that the `ArenaManager` would maybe need a cleanup action, but since it's an in-memory structure that doesn't really make sense